src/main/java/com/wavefront/generators/TraceGenerator.java

```diff
@@ -96,8 +104,18 @@ protected void startGeneration(boolean isRealTime, @Nonnull GeneratorConfig gene
         }
       }
     }
+    logger.info("Generation complete!\n" + ANSI_YELLOW + String.format(generatorConfig.getGeneratorConfigFile() +
```

Comment:
As far as I remember getGeneratorConfigFile() may return null, in case generation parameters were provided through the command line. Please check this case.